napalm_junos/tests/base.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```

Comment:
I don't understand why would you insert this hashbang? Is this going to be used like a script?